packages/color/package.json

```diff
@@ -11,6 +13,9 @@
     "@theme-ui/css": "^0.3.1",
     "polished": "^3.4.1"
   },
+  "peerDependencies": {
```

Comment:
I think we should come to a consensus here whether to put the dependencies for types in peerDep, devDep or Dep. I opted for devDep: https://github.com/system-ui/theme-ui/pull/671/files#diff-03713682702149e044f8824251410926R22

Reply:
yeah, previously the packages depended on _common concepts_.
These concepts are now described as types and we keep them in `@theme-ui/core`, so the dependency needs to be noted in package.json.

I think there are two possibilities:

- `@theme-ui/color` has `@theme-ui/core` both in peer dependencies and a dev dependencies.
  - Dev dep only would be a TypeScript build error for users without any previous warning. This is also how I got _"semantic error TS2307 Cannot find module '@theme-ui/core'."_ on Circle CI :D
- `@theme-ui/color` depends on `@theme-ui/core`. It should be okay in monorepo with the same versions. I think I'll go this way since `color` already depends on `css`.

After the build there is no import to `@theme-ui/core` so it won't get bundled for people who use only color utils.